crates/viewer/re_renderer/shader/world_grid.wgsl

```diff
@@ -0,0 +1,154 @@
+#import <./global_bindings.wgsl>
+
+struct WorldGridUniformBuffer {
+    color: vec4f,
+
+    /// A value of [`super::GridPlane`]
+    orientation: u32,
+
+    /// How far apart the closest sets of lines are.
+    spacing: f32,
+
+    /// How thick the lines are in UI units.
+    thickness_ui: f32,
+}
+
+@group(1) @binding(0)
+var<uniform> config: WorldGridUniformBuffer;
+
+// See world_grid::GridPlane
+const ORIENTATION_XZ: u32 = 0;
+const ORIENTATION_YZ: u32 = 1;
+const ORIENTATION_XY: u32 = 2;
+
+struct VertexOutput {
+    @builtin(position)
+    position: vec4f,
+
+    @location(0)
+    scaled_world_plane_position: vec2f,
+};
+
+const PLANE_SCALE: f32 = 10000.0;
```

Comment:
![meanig](https://github.com/user-attachments/assets/3caa76d0-fd7a-4ed2-b2f7-a7acb6c8ff56)

Is this the number of grid lines in each direction or somesuch?